src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioSymbolNavigationService.cs

```diff
@@ -82,6 +84,31 @@ public bool TryNavigateToSymbol(ISymbol symbol, Project project, CancellationTok
                 return false;
             }
 
+            // Should we prefer navigating to the Object Browser over metadata-as-source?
+            if (options.GetOption(VisualStudioNavigationOptions.NavigateToObjectBrowser, project.Language))
+            {
+                var libraryService = project.LanguageServices.GetService<ILibraryService>();
+                if (libraryService == null)
+                {
+                    return false;
+                }
+
+                var compilation = project.GetCompilationAsync(cancellationToken).WaitAndGetResult(cancellationToken);
+                var navInfo = libraryService.NavInfo.CreateForSymbol(symbol, project, compilation);
+                if (navInfo == null)
+                {
+                    navInfo = libraryService.NavInfo.CreateForProject(project);
+                }
+
+                if (navInfo == null)
+                {
+                    return false;
```

Comment:
Yeah, I suppose. I was trying to avoid that. Although, really, this'll never get hit. We'll always at least try to go to the project. So, I'll add the fallback in the code, but it shouldn't ever happen.